google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/SubscriberClientSnippets.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** This class contains snippets for the {@link SubscriberClient} interface. */
+
+package com.google.cloud.examples.pubsub.snippets;
+
+import com.google.cloud.Identity;
+import com.google.cloud.Role;
+import com.google.cloud.pubsub.spi.v1.PagedResponseWrappers.ListSubscriptionsPagedResponse;
+import com.google.cloud.pubsub.spi.v1.PublisherClient;
+import com.google.cloud.pubsub.spi.v1.SubscriberClient;
+import com.google.iam.v1.Binding;
+import com.google.iam.v1.Policy;
+import com.google.iam.v1.TestIamPermissionsResponse;
+import com.google.pubsub.v1.ListSubscriptionsRequest;
+import com.google.pubsub.v1.ProjectName;
+import com.google.pubsub.v1.PullResponse;
+import com.google.pubsub.v1.PushConfig;
+import com.google.pubsub.v1.ReceivedMessage;
+import com.google.pubsub.v1.Subscription;
+import com.google.pubsub.v1.SubscriptionName;
+import com.google.pubsub.v1.TopicName;
+import edu.emory.mathcs.backport.java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+/** This class contains a number of snippets for the {@link SubscriberClient} interface. */
+public class SubscriberClientSnippets {
+
+  private final String projectId;
+
+  public SubscriberClientSnippets(String projectId) {
+    this.projectId = projectId;
+  }
+
+  /** Example of creating a pull subscription for a topic. */
+  public Subscription createSubscription(String topic, String mySubscription) throws Exception {
```

Comment:
I think `subscriptionId` might be a more appropriate name than `mySubscription`.